cmake/modules/VersionInfo.cmake

```diff
@@ -1,6 +1,6 @@
 FIND_PACKAGE(Git)
 IF(GIT_FOUND AND NOT FORCE_VERSION)
-	# Look for git tag information (e.g. Stable: "v1.0.0", Non-stable: "v1.0.0-123-a1b2c3d4")
+	# Look for git tag information (e.g. Tagged: "v1.0.0", Non-tagged: "v1.0.0-123-a1b2c3d4")
```

Comment:
`v1.0.0-123-a1b2c3d4` is the output of `git describe`, but it should be `v1.0.0-123-a1b2c3d`.